base/regex.jl

```diff
@@ -410,9 +431,9 @@ If `overlap=true`, the matching sequences are allowed to overlap indices in the
 original string, otherwise they must be from disjoint character ranges.
 
 !!! compat "Julia 1.3"
-     This method requires at least Julia 1.3.
+     This method requires at least Julia 1.3; using a character as the pattern requires Julia 1.6+.
```

Comment:
Add a separate `!!! compat "Julia 1.6"`, I think?

Reply:
That one was added by StefanKarpinski, I would not know what they prefer.

Reply:
Separate is probably better. Sorry about that!